core/templates/components/state-directives/outcome-editor/outcome-editor.component.ts

```diff
@@ -235,6 +268,14 @@ export class OutcomeEditorComponent implements OnInit {
     this.destinationEditorIsOpen = false;
     this.correctnessLabelEditorIsOpen = false;
     this.savedOutcome = cloneDeep(this.outcome);
+
+    this.onMobile = (
+      this.windowDimensionsService.getWidth() <= 500);
+    this.resizeSubscription = this.windowDimensionsService.getResizeEvent()
+      .subscribe(event => {
+        this.onMobile = (
+          this.windowDimensionsService.getWidth() <= 500);
+      });
```

Comment:
Should these dimensions (500) be kept in sync with some dimensions in the CSS files?

Reply:
Yes, I've added the comments to make this clear.